packages/app/src/templates/reducers/template.interface.ts

```diff
@@ -3,9 +3,26 @@
  * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
  */
 
+import { PayloadAction } from "@reduxjs/toolkit"
+import { TemplatesEvent } from "App/templates/constants"
+import { CreateTemplateError } from "App/templates/errors"
 import { Template } from "App/templates/dto"
 
 export interface TemplateState {
   data: Template[]
+  loading: boolean
+  loaded: boolean
```

Comment:
Do we really need separate loaded and loading state? maybe we can add a state `addingNewTemplateState` which will be enum like
`export enum addingNewTemplateState {
  Loading,
  Loaded,
  Empty,
  Error,
}`
Current names can be confusing because there will be more loading states (for example for editing and deleting).

Reply:
Currently, we will not support two types of action in MC. If you, for example, add a new record, you can't edit anything simultaneously, so this action should be locked.